docs/content/guides/operations/providers/aws-provider/howto-aws-provider-irsa/index.md

```diff
@@ -0,0 +1,130 @@
+---
+type: docs
+title: "How-To: Configure the AWS cloud provider with IAM Roles for Service Accounts (IRSA)"
+linkTitle: "AWS provider with IAM Roles for Service Accounts(IRSA)"
+description: "Learn how to configure the AWS provider with IAM Roles for Service Accounts(IRSA) for your Radius Environment"
+weight: 300
+categories: "How-To"
+tags: ["AWS"]
+---
+
+The AWS provider allows you to deploy and connect to AWS resources from a Radius Environment on an EKS cluster. It can be configured:
+
+- [Interactively via `rad init`](#interactive-configuration)
+- [Manually via `rad env update` and `rad credential register`](#manual-configuration)
+
+## Prerequisites
+
```

Comment:
I think we should add `aws configure` as a pre-req in both the AWS provider how-to's since we changed the flow to pick the account from local aws config

Reply:
added it